frontend/packages/dev-console/src/components/helm/HelmInstallUpgradePage.tsx

```diff
@@ -55,18 +56,24 @@ const HelmInstallUpgradePage: React.FunctionComponent<HelmInstallUpgradePageProp
       let res;
       try {
         res = await coFetchJSON(config.helmReleaseApi);
-      } catch {
-        if (ignore) return;
-      }
+      } catch {} // eslint-disable-line no-empty
```

Comment:
shall we log in console as `error/warning`

Reply:
No need i think as we are going to ignore that anyways. We do handle cases gracefully if no data is loaded. And fetch throws an error that is visible on the console so no need to add another console log statement.